x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/esql/search_filter.cy.ts

```diff
@@ -68,8 +69,8 @@ describe(
       updateDateRangeInLocalDatePickers(DISCOVER_CONTAINER, INITIAL_START_DATE, INITIAL_END_DATE);
       waitForDiscoverFieldsToLoad();
       cy.get(DISCOVER_CONTAINER).within(() => {
```

Comment:
This container already limits the search inside a particular container. Do you think it makes sense to use this or add `tabId` as second param?

I am still debating and i think i will be okay with both.

Reply:
i think tabId is preferrable, that way future users of screen/task helpers can just use them outright, and not need to know to wrap within .within, otherwise risk creating flaky tests